src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/conditionalorder/SystemRedoPronouncedCoverLettersTask.java

```diff
@@ -56,7 +56,7 @@ public class SystemRedoPronouncedCoverLettersTask implements Runnable {
     @Autowired
     private AuthTokenGenerator authTokenGenerator;
 
-    public static final String NOTIFICATION_FLAG = "coPronouncedCoverLetterResent";
+    public static final String NOTIFICATION_FLAG = "coPronouncedConfidentialCoverLetterResent";
```

Comment:
Out of interest, how come we need to change the notification flag? I'd thought we didn't make changes for those cases in the last run and the previous flag would still be false for them

Reply:
It's for when there would be one confidential applicant, then the case would have run on the normal doc but not the other one so needs to run again.